packages/contracts-bedrock/src/safe/DeputyPauseModule.sol

```diff
@@ -71,44 +68,47 @@ contract DeputyPauseModule is ISemver, EIP712 {
     ISuperchainConfig internal immutable SUPERCHAIN_CONFIG;
 
     /// @notice Typehash for the Pause action.
-    bytes32 internal constant PAUSE_MESSAGE_TYPEHASH = keccak256("PauseMessage(bytes32 nonce)");
+    bytes32 internal constant PAUSE_MESSAGE_TYPEHASH = keccak256("PauseMessage(bytes32 nonce,address identifier)");
 
     /// @notice Typehash for the DeputyAuth message.
     bytes32 internal constant DEPUTY_AUTH_MESSAGE_TYPEHASH = keccak256("DeputyAuthMessage(address deputy)");
 
     /// @notice Address of the Deputy account.
     address public deputy;
 
-    /// @notice Address of the DeputyGuardianModule used by the SC Safe.
-    IDeputyGuardianModule public deputyGuardianModule;
-
     /// @notice Used nonces.
     mapping(bytes32 => bool) public usedNonces;
 
     /// @notice Semantic version.
-    /// @custom:semver 1.0.0-beta.2
-    string public constant version = "1.0.0-beta.2";
+    /// @custom:semver 2.0.0-beta.0
+    string public constant version = "2.0.0-beta.0";
 
```

Comment:
This contract is a bit weird since its not part of the OPCM upgrade path. Is the plan to remove the `-beta.0` tag before or after it goes to governance?

Reply:
Oh we should remove -beta.0